### PR TITLE
make search box placeholder more specific

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Type

🐛 Bugfix  

- [x] Demoed to @Heesoomoon 

## Value
Makes the search bar more specific
Fixes a eslint bug

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "eslint-config-wellcome": "^1.0.8",
```
Comment (davidpmccormick) [inline package.json]: Shall we kill this and add

```
module.exports = {
  extends: ['standard'],
  rules: {
    'space-before-function-paren': 'off',
    'no-return-assign': 'off',
    semi: ['error', 'always']
  }
};
```

to our own eslintconfig?

Comment (jamesgorrie) [inline package.json]: Yip - move to another PR?
